include/glow/ExecutionContext/ExecutionContext.h

```diff
@@ -38,7 +36,7 @@ class DeviceBindings {
     return llvm::make_unique<DeviceBindings>(backend_);
   }
 
-  llvm::StringRef getBackendKind() { return backend_; }
+  llvm::StringRef getBackendName() { return backend_; }
```

Comment:
there are no related changes associated with this in the PR, probably an indication of method not used.

Subject to a diff PR probably, but probably can remove this all together.